opennms-full-assembly/pom.xml

```diff
@@ -348,6 +348,8 @@
                 <!-- TODO: Should we import both versions of Jetty? -->
                 <feature>jetty/8.1.14.v20131031</feature>
                 <feature>jetty/9.3.9.v20160517</feature>
+                <!-- TODO: Needed? -->
+                <feature>jetty/9.4.18.v20190429</feature>
```

Comment:
I added the new version of Jetty referenced by Karaf 4.2.6 but I'm not sure if that is needed, should I be replacing one of the versions above?

Reply:
AFAIK, we should not need the jetty features, maybe it's dependencies.

Reply:
Ok, it seems to work without those added. So I've removed them in latest commit.